make.go

```diff
@@ -991,6 +991,7 @@ func execMake(args []string, usage func()) {
 	fmt.Printf("    dh-make-golang create-salsa-project %s\n", debsrc)
 	fmt.Printf("\n")
 	fmt.Printf("Once you are happy with your packaging, push it to salsa using:\n")
+	fmt.Printf("    git push origin %s\n", debBranch)
 	fmt.Printf("    gbp push\n")
```

Comment:
Looks good to me, however I'm still not sure about `gbp push`. As per the [man page](https://manpages.debian.org/buster/git-buildpackage/gbp-push.1.en.html) and the [bug report](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=908204#10), it is clearly indicated that `gbp push` is intended to be run *after uploading a Debian package to the archive*. In the bug report Guido suggest simply running `git push` when there was no release.

Adding a `--tips` flag to gbp push has been suggested [here](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=908204#42) but looks not implemented atm.

Personally, I'm doing a `git push --all && git push  --tags` in order to push all the branches and the tags to the Salsa repository. I wonder what the rest of the team think about that. @zhsj any opinion on this?

Reply:
It's clear that the `gbp push` workflow doesn't work here. However I would much prefer implementing a `dh-make-golang push` command. And I don't like `git push --tags` since it pushes all upstream tags which are not needed, for example upstream have v1.0.0 -> v10.0.0 tags, while we only need upstream/10.0.0 tag.

Reply:
Ack, so this `dh-make-golang push` command should be running a `git push origin` to push the branches and then push all the tags beginning by `upstream/` ?

Reply:
Probably like `gbp push`, but allow to push wip debian branch: 

+ `git push origin debian/sid`
+ `git push origin upstream`
+ `git push origin upstream/<tag>`
+ (optional) `git push origin pristine-tar`
+ (optional) `git push origin debian/<tag>`

While I'm writing above, I find it forces `dh-make-golang` to parse `gbp.conf` to see the `upstream` branch name ..